src/phpDocumentor/Reflection/Php/Factory/File.php

```diff
@@ -66,22 +63,22 @@ public function __construct(NodesFactory $nodesFactory, $middleware = array())
     /**
      * Returns true when the strategy is able to handle the object.
      *
-     * @param string $file path to check.
```

Comment:
The reason why string was specified here was because this method should only accept a string. This helped me with some autocompletion in my ide.
This comment applies to all match methods of all factories.

Reply:
Hmmm... so all calls to all the various `matches()` methods should always return `false`, because the *strings* passed in should **never** `instanceof` to `true` for the classnames being checked... ?